src/avt_camera/scripts/mono_camera.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+from pymba import *
+import rospy
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+from time import sleep
+import numpy as np
+import sys
+class MonoCamera:
+    TOPIC_NAME = 'mono_camera/raw'
+    NODE_NAME = 'mono_camera'
+    
+
+    def __init__(self):
+        rospy.init_node(self.NODE_NAME)
+        self._pub = rospy.Publisher(self.TOPIC_NAME, Image, queue_size=10)
+        self._bridge = CvBridge()
+        self._camera_id = rospy.get_param('~camera_id', None)
+
+    def run(self):
+        
+        with Vimba() as vimba:
+            system = vimba.getSystem()
+            system.runFeatureCommand("GeVDiscoveryAllOnce")
+            rospy.sleep(0.1)
+
+            camera_ids = vimba.getCameraIds()
+            if not camera_ids:
+                rospy.logerr("No cameras found, big RIP")
+                sys.exit(0)
+
```

Comment:
Somehow check if the same camera is requested and give a reasonable error message if that happened

Reply:
neil: "try catch on opencamera"